templates/bundles/DemosPlanCoreBundle/DemosPlanCore/includes/base_footer.html.twig

```diff
@@ -7,33 +7,38 @@
 
             {% if hasPermission('area_main_static_documents') %}
                 <a class="c-footer__link" href="{{ path("DemosPlan_misccontent_static_documents") }}"
+                   data-cy="static_documents"
```

Comment:
Some time ago we discussed naming for those hooks and came up with some kind of namespacing "per component", to make tests more readable. for those footer links, i'd suggest

```suggestion
                   data-cy="footerNav:staticDocuments"
```
or the like. The other changes have to be adapted then, too.